docs: document ES|QL Fast mode toggle for approximation (#7141) (#7143)
elastic/docs-content: explore-analyze/discover/try-esql.md

When your {{esql}} query uses a [`STATS BY`](elasticsearch://reference/query-languages/esql/commands/stats-by.md) clause with a single grouping field, **Discover** displays the results as expandable groups instead of a flat table. Each row represents one unique value of the grouping field, and you can expand it to inspect the underlying documents without leaving the query. The results count above the table reports the number of groups instead of the number of documents.

:::{note}
:applies_to: {"stack": "preview 9.5", "serverless": "preview"}

> On large datasets, you can return faster, estimated results for a query that uses exactly one `STATS` command. Select the {icon}`bolt` Fast mode button next to the query bar. Refer to [Get faster results with Fast mode](/explore-analyze/query-filter/languages/esql-kibana.md#approximation-fast-mode).

Author: ```suggestion
When searching large datasets, you can get faster, estimated results by using {icon}`bolt` **Fast mode**. Refer to [](/explore-analyze/query-filter/languages/esql-kibana.md#approximation-fast-mode).
```
